src/main/java/com/stacksurveyor/backend/controllers/RegisterForm.java

```diff
@@ -0,0 +1,25 @@
+//	Copyright (C) 2021  StackSurveyor
+//
+//	This program is free software: you can redistribute it and/or modify
+//	it under the terms of the GNU General Public License as published by
+//	the Free Software Foundation, either version 3 of the License, or
+//	(at your option) any later version.
+//
+//	This program is distributed in the hope that it will be useful,
+//	but WITHOUT ANY WARRANTY; without even the implied warranty of
+//	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//	GNU General Public License for more details.
+//
+//	You should have received a copy of the GNU General Public License
+//	along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+package com.stacksurveyor.backend.controllers;
```

Comment:
Don't put this form input in the controllers package. The controllers package is for all the API endpoints, not form inputs. Either make a package or a separate class for this, don't use the controllers package